integration-tests/cypress/e2e/project/deck/deck.cy.ts

```diff
@@ -0,0 +1,244 @@
+/*******************************************************************************
+ * Copyright (c) 2024  Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { Project } from '../../../pages/Project';
+import { Projects } from '../../../pages/Projects';
+import { isCreateProjectFromTemplateSuccessPayload } from '../../../support/server/createProjectFromTemplateCommand';
+import { Explorer } from '../../../workbench/Explorer';
+
+describe('Verify the Deck Representation', () => {
+  let studioProjectId: string = '';
+  let taskProjectId: string = '';
+  before(() => {
+    // We create the Deck View from the stereotype before executing the tests
+    new Projects().visit();
+    cy.createProjectFromTemplate('blank-studio-template').then((res) => {
+      const payload = res.body.data.createProjectFromTemplate;
+      if (isCreateProjectFromTemplateSuccessPayload(payload)) {
+        studioProjectId = payload.project.id;
+
+        new Project().visit(studioProjectId);
+        cy.getByTestId('new-model').click();
+        cy.getByTestId('name-input').type('Deck View');
+        cy.getByTestId('stereotype').click();
+        cy.get('li').filter(':contains("Deck View")').click();
+        cy.getByTestId('create-document').click();
+      }
+    });
+  });
+  after(() => {
+    // We delete the created studio once all tests have been executed
+    cy.deleteProject(studioProjectId);
+  });
+
+  beforeEach(() => {
+    // We create the Task project from the template before each tests.
+    new Projects().visit();
+    cy.createProjectFromTemplate('task-template').then((res) => {
+      const payload = res.body.data.createProjectFromTemplate;
+      if (isCreateProjectFromTemplateSuccessPayload(payload)) {
+        taskProjectId = payload.project.id;
+        new Project().visit(taskProjectId);
+      }
+    });
+  });
```

Comment:
This could be moved inside the nex context and declared as before to only create one task model and one representation instead of create one for each test

Reply:
done; I use the same Deck rep and the same task project for each context. I could use only one for the whole deck.cy.test but it will be more difficult to isolate them in the future.